common/xembed.c

```diff
@@ -24,6 +24,9 @@
 #include "common/util.h"
 #include "common/atoms.h"
 
+/* I should really include the correct header instead... */
```

Comment:
Nope. At least I call myself out for this uglyness. ;-)

The problem is that the stuff in `common/` should be self-contained and this dependency goes against that and i have no good idea what to do about it.